HHS/Head-Start-TTADP
# [PROD] 04/25 [TTAHUB-2871] Fix for duplicating goals and objectives from RTR

## Description of change
**TTAHUB-2871**: Fix bug that caused goals and objectives to duplicate from RTR.
**TTAHUB-2573, 2575, 2576, 2577**: Use DOMPurify ([github page](https://github.com/cure53/DOMPurify)) ([npm page](https://www.npmjs.com/package/dompurify)) to sanitize rich fields.

## How to test
**TTAHUB-2871**: Create a goal and objective from RTR, edit the goal and change the objective title (the goal and obj should exist once).
**TTAHUB-2573, 2575, 2576, 2577**: Put a bunch of formatted HTML into one of the rich text fields. Save to the database. Ensure that the stored data is completely sanitized. Reload the rich text field, and validate that what is returned to the client doesn't include <, >, etc.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2871


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
